libflux/flux/src/lib.rs

```diff
@@ -1118,22 +1118,39 @@ from(bucket: v.bucket)
     }
 
     #[test]
-    //CSV package's docs are accessed from their specific index in the vector of DocPackages, ensuring that they are accessible and correct
-    // XXX: rockstar (5 Aug 2021) - This test can be flaky. For example, there was a
-    // failure because another patch was merged that added a new stdlib, which pushed
-    // `csv` from the 19th item to the 20th item.
```

Comment:
@rockstar thank you for your comments, the tests in this PR cannot pass for the reasons you explained. An introduction of any "contrib/*" package (in this PR) increments a position of `csv`package doc. I fixed the doc tests to 'find by name' herein. I did not find any place where a specific index `20` is required.

Reply:
The flaky test were repaired meanwhile in 5b821eef54c6610d7ca1e321ad702bdba64f7495